integrations.yml

```diff
@@ -58,3 +58,9 @@
   website: "https://eazybi.com/integrations/basecamp"
   icon: "eazybi.png"
   category: 3
+- 
+  name: "AssessTEAM"
+  description: "Continuous feedback based employee evaluation system & project profitability reporting suite"
+  website: "https://www.assessteam.com"
+  icon: "Assessteam-icon256.png"
```

Comment:
Hey @ps97-ketangohil, you'll need to add the icon in a commit on the same branch as this pull request.